crates/ty_python_semantic/src/site_packages.rs

```diff
@@ -544,6 +613,8 @@ impl PartialEq<SystemPathBuf> for PythonHomePath {
 
 #[cfg(test)]
 mod tests {
+    use core::panic;
+
```

Comment:
this is part of the `prelude`

```suggestion
```

Reply:
No clue why the editor added that / why Clippy wouldn't remove it?

Reply:
(I'll remove this before merging, but won't update it until then because I'll need to rebase the whole stack)